packages/flutter/test/widgets/widget_inspector_test_utils.dart

```diff
@@ -12,7 +12,9 @@ import 'package:flutter_test/flutter_test.dart';
 class TestWidgetInspectorService extends Object with WidgetInspectorService {
   final Map<String, ServiceExtensionCallback> extensions = <String, ServiceExtensionCallback>{};
 
-  final Map<String, List<Map<Object, Object?>>> eventsDispatched = <String, List<Map<Object, Object?>>>{};
+  final Map<String, Map<String,List<Map<Object, Object?>>>> eventsDispatched
+    = <String, Map<String, List<Map<Object, Object?>>>>{};
+  final  List<Object?> objectsInspected = <Object?>[];
```

Comment:
proper formatting:

```
final Map<String, Map<String, List<Map<Object, Object?>>>> eventsDispatched =
    <String, Map<String, List<Map<Object, Object?>>>>{};
```

tip: to get the formatting right for flutter/flutter, I sometimes take a snippet of code like this and paste into another project that uses dart format (i.e. devtools), format the code, and then bring back to here.

Reply:
I don't know if it helps here, but FWIW there's a "Format Selection" in VS Code. It basically runs the formatter on the file, compares the ASTs and finds the changes that are within the selection and applies them. This means it will provide the correct indentation for code nested inside classes/blocks even if those classes/blocks aren't being formatted.

(If you use the Git support in VS Code, there's also a way to make it format only git-modified lines when you do a format, but I don't use it so I can't be sure how well it works.. I'd be interested in bug reports if it doesn't work well though!)